packages/core/test/exceptions/exceptions-handler.spec.ts

```diff
@@ -94,36 +93,36 @@ describe('ExceptionsHandler', () => {
           new ExecutionContextHost([0, response]),
         );
 
-        expect(statusStub.calledWith(status)).to.be.true;
-        expect(jsonStub.calledWith({ message, statusCode: status })).to.be.true;
+        expect(statusStub.calledWith(status)).toBeTruthy()
+        expect(jsonStub.calledWith({ message, statusCode: status })).toBeTruthy();
       });
     });
     describe('when "invokeCustomFilters" returns true', () => {
       beforeEach(() => {
         sinon.stub(handler, 'invokeCustomFilters').returns(true);
       });
       it('should not call status and json stubs', () => {
-        expect(statusStub.notCalled).to.be.true;
-        expect(jsonStub.notCalled).to.be.true;
+        expect(statusStub.notCalled).toBeTruthy()
```

Comment:
```suggestion
        handler.next(new Error(), jest.fn() as any)
        
        expect(statusStub.notCalled).toBeTruthy()
```

I just notice that the current version didn't test anything. See:

https://github.com/nestjs/nest/blob/28516d739df350564eceea546dd1f58b5e9c9800/packages/core/test/exceptions/exceptions-handler.spec.ts#L101-L109